index.bs

```diff
@@ -9733,7 +9733,9 @@ various channel configurations can be achieved.
 	{{AudioWorkletNodeOptions/numberOfOutputs}} = 1
 :: If {{AudioWorkletNodeOptions/outputChannelCount}} is
 	unspecified, the output channel count will match
-	<a>computedNumberOfChannels</a> from the input.
+	<a>computedNumberOfChannels</a> from the input. If
+	{{AudioWorkletNodeOptions/outputChannelCount}} is specified
+	with one value, use the value to configure the output channel.
 
 : All other cases
 :: If {{AudioWorkletNodeOptions/outputChannelCount}} is
```

Comment:
Maybe we should add an additional sentence here that if outputChannelCount is specified, the values are used as given? (Not really part of the issue, but it's a useful related clarification.)